docs/assets/js/ui.js

```diff
@@ -160,9 +160,40 @@ function initNotesToggles() {
 	});
 }
 
+function initFeatureFilter() {
+	var filterInput = document.getElementById('filter-features');
+	var featuresRows;
+
+	if (filterInput) {
+		featuresRows = document.querySelectorAll('[data-feature-name]');
```

Comment:
Also I'd consider building a simple object map to allow faster lookups, so you store this:

```
{
   "array.map": [element],
   "fetch": [element],
   ...
}
```